recipes/bellmans-gapc/nussinov.gap

```diff
@@ -0,0 +1,151 @@
+
+signature Nuss(alphabet, answer) {
+
+  answer nil(void);
+  answer right(answer, alphabet);
+  answer pair(alphabet, answer, alphabet);
+  answer split(answer, answer);
+  choice [answer] h([answer]);
+
```

Comment:
I would remove this entire file, it is not needed without the extended tests, right?

Reply:
agreed. File is gone.